components/camel-dynamic-router/pom.xml

```diff
@@ -40,8 +41,24 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-core-processor</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>jakarta.validation</groupId>
+      <artifactId>jakarta.validation-api</artifactId>
+      <version>3.0.2</version>
```

Comment:
Do not use hardcoded versions

Reply:
I probably do not need the validation annotation that I was using, so I'll remove it.  For the future, do you prefer the use of a property for the version, or something else?

Reply:
I got rid of the dependencies that had hard coded version so these are gone, too.